Keep signingConfig commented out by default?

nahuel-soldevilla opened the pull request:
# Description

Is is possible to keep the signingConfig commented out by default? 

# How Has This Been Tested?

Debugged using android studio.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to documentation
- [x] My changes generate no new warnings

trife commented:
Is there a need to do this instead of just compiling with the `debug` build variant?

nahuel-soldevilla commented:
Thanks @trife, yeah that was basically what I needed, It seems somehow my config was not using any of those variants and there was no warning when I launched the app, but now that I changed it to debug and change it back to release I do see a clear warning:

![android-studio-debug-warning](https://user-images.githubusercontent.com/19394293/121461604-930f0300-c985-11eb-9b0e-d9b6f0427059.png)

So I changed it to debug and now it's working fine. It seems the `keystore.config` file should be there even for debug, so I created an empty file.
```
Executing tasks: [:app:assembleDebug] in project KSU-FieldBook\Field-Book


FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file 'KSU-FieldBook\Field-Book\app\build.gradle' line: 10

* What went wrong:
A problem occurred evaluating project ':app'.
> KSU-FieldBook\Field-Book\app\keystore.config (The system cannot find the file specified)
```

trife commented:
I made some changes that should remove the necessity of creating the empty file that I'll push in the next couple days.
